internal/util/paramtable/component_param.go

```diff
@@ -801,6 +803,15 @@ func (p *queryNodeConfig) initSmallIndexParams() {
 	}
 }
 
+func (p *queryNodeConfig) initLoadMemoryUsageFactor() {
+	loadMemoryUsageFactor := p.Base.LoadWithDefault("queryNode.loadMemoryUsageFactor", "3")
```

Comment:
NIT: Magic "Number", maybe use const `DEFAULAT_LOAD_MEMORY_FACTOR` in the future.